website_event_sale_hide_ticket/models/event_ticket.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class EventEventTicket(models.Model):
+    _inherit = "event.event.ticket"
+
+    show_in_website = fields.Boolean(default=True)
```

Comment:
I think `website_published` is more consistent with odoo naming, as you can find this field in many website related models already